CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+
+A great way to make sure we cover relevant areas of the Cardano protocol first, is to understand what _you_ are trying to build or understand _right now_.
+
+We use [Q&A discussions](https://github.com/cardano-scaling/cardano-blueprint/discussions/new?category=q-a) to gather immediate questions and use them to identify gaps for where blueprints can help most.
+
+While discussions and providing answers in the [Q&A forum](https://github.com/cardano-scaling/cardano-blueprint/discussions/categories/q-a) are obviously welcome, ideally we would be [writing blueprints](#write-blueprints) and be able to answer by pointing future readers there.
+
+### Share feedback and ideas
+
+If you happen to find the information provided through blueprints _useful_ to your current project, or if you are confused about the _why_ of blueprints, the [general category](https://github.com/cardano-scaling/cardano-blueprint/discussions/categories/general) is the right place for that.
+
+Or maybe you have an idea on how we could reach our [goals](https://cardano-scaling.github.io/cardano-blueprint/#goal) by changing one or the other process? Ideas are collected in the [ideas category](https://github.com/cardano-scaling/cardano-blueprint/discussions/categories/ideas).
+
+We would love to hear about any feedback! 💙
+
+### Report issues
+
+[Submit an issue](https://github.com/cardano-scaling/cardano-blueprint/issues/new) if you spot an errors, or should you encounter a problem with using or contributing to the blueprints themselves.
+
+## Write blueprints
+
+Creating or updating blueprints is the bread & butter of this project and anyone is invited to do just that!
+
+The bandwidth of possible contributions is very broad and can range from fixing typos via the `Suggest an edit` <i class="fa fa-edit"></i> button on the [rendered website](https://cardano-scaling.github.io/cardano-blueprint), over contributing a whole set of documents about how consensus works in Cardano, to including a conformance test suite.
+
+In fact, it's not even set in stone [what a blueprint is](https://cardano-scaling.github.io/cardano-blueprint/#what-is-a-blueprint) and we are gladly exploring any asset that you think covers our values and contributes to reaching the goal of the Cardano Blueprint initiative.
+
+### Creating a pull request
+
+Thank you for contributing your changes by opening a pull request!
+
+We do appreciate it if your pull request meets the following criteria:
+
++ **Description of the changes**: providing a clear summary of the changes is beneficial
++ **Quality of changes**: ensure that new or updated automated tests validate your changes
++ **Scope**: we prefer multiple pull requests that are well-scoped rather than a single large one
++ **Correctness**: the pull request should pass the continuous integration (CI) pipeline.
+
+The project is currently in an expand-and-gather phase with no requirement of approvals, but consider requesting for reviews on Github-suggested reviewers.
```

Comment:
Should there be some process of approval by knowledgeable people? Something like codeowners. I think at least Consensus we would like to review PRs that target the Consensus side of the blueprint.

Reply:
Are you prepared to review any contribution at this stage and the time this requires? This section does not mean that you don't get a chance to chime in as I will definitely ask for your review (being the author of big sections now). But it rather means that we are not intending to gate keep even before we have multiple groups of people engaged.

I just don't want to limit creativity and contributions by increasing processes too much right away? And on a side note, code owners is a premium feature of github :sweat_smile:

Reply:
@jasagredo see also this discussion: https://github.com/cardano-scaling/cardano-blueprint/discussions/14

I would probably keep that open until we have something in place (once we leave the "expand-and-gather" phase)

Reply:
> The project is currently in an expand-and-gather phase with no requirement of approvals

I would say we can already add the consensus team to the code owners of the consensus relevant bits :+1:

Reply:
https://github.com/cardano-scaling/cardano-blueprint/issues/23